src/NodaTime.Test/Text/LocalDatePatternTest.cs

```diff
@@ -236,12 +236,15 @@ public void BclLongDatePatternGivesSameResultsInNoda(CultureInfo culture)
                 return;
             }
 
-            // We got year-of-era and year the wrong way round in 1.x.
+            // We got year-of-era and year wrong in 1.x.
             // See https://github.com/nodatime/nodatime/issues/374
-            // It would be a breaking change to make the 2.0 change in 1.x; we could
-            // potentially change the behaviour to allow the given cultures to work
-            // by interpreting YYYY as yyyy if there's no era specifier, but for the moment
-            // these are just expected failures :(
+            // Basically, in 1.x we treat "y" as absolute year and "Y" as year-of-era.
+            // In 2.0 we treat "y" as year-of-era and "u" as absolute year.
+            // In the "known broken cultures" the format contains an era specifier (g)
+            // which we deem to be invalid in 1.x due to the lack of a year-of-era specifier.
```

Comment:
Slightly confusing: it _does_ have a year-of-era specifier (`y`), just that we think that's an absolute year specifier.